src/badguy/dispenser.cpp

```diff
@@ -267,20 +275,24 @@ Dispenser::launch_badguy()
       return;
 
   //FIXME: Does is_offscreen() work right here?
-  if (!is_offscreen() && !Editor::is_active()) {
+  if (!is_offscreen() && !Editor::is_active())
+  {
     Direction launchdir = m_dir;
-    if ( !m_autotarget && m_start_dir == Direction::AUTO ){
+    if ( !m_autotarget && m_start_dir == Direction::AUTO )
+    {
       Player* player = get_nearest_player();
-      if ( player ){
+      if ( player )
```

Comment:
(Those ones aren't your fault, but if you're passing around that code and feel motivated, the spaces inside the parentheses...)

Reply:
Yes, I will do that (and all of the above). There was so much code mess in `dispenser.cpp` that I missed something.